meta-resin-common/recipes-support/resin-device-register/resin-device-register/resin-device-register

```diff
@@ -53,13 +53,14 @@ while true; do
         fi
 
         status_code=$(curl -s -X POST -w "%{http_code}" \
+                    --header "Authorization: Bearer $PROVISIONING_API_KEY" \
                     --output $LOG_FILE \
                     --data-urlencode "user=$USER_ID" \
                     --data-urlencode "application=$APPLICATION_ID" \
                     --data-urlencode "device_type=$DEVICE_TYPE" \
                     --data-urlencode "uuid=$UUID" \
                     --data-urlencode "api_key=$DEVICE_API_KEY" \
-                    "$API_ENDPOINT/device/register?apikey=$PROVISIONING_API_KEY")
+                    "$API_ENDPOINT/v4/device/register")
```

Comment:
As discussed, no!! This moves from the custom endpoint `/device/register` to `/v4/device/register` which seems to be ingnored and converted to `/v4/device`, but also doesn't use the custom transaction setup for creating deviceApiKeys which are linked to newly provisioned devices.